learners/setup.md

```diff
@@ -2,53 +2,136 @@
+
+Download
+[00001_01.ome.tiff](
+https://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/MitoCheck/00001_01.ome.tiff)
+ ([CC BY 4.0 Licence](
+https://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/MitoCheck/readme.txt))
+ from the open microscopy environment pages to your working directory.
+
+Download
+[Plate1-Blue-A-12-Scene-3-P3-F2-03.czi](
+https://downloads.openmicroscopy.org/images/Zeiss-CZI/idr0011/Plate1-Blue-A_TS-Stinger/Plate1-Blue-A-12-Scene-3-P3-F2-03.czi)
+ ([CC BY 4.0 Licence](
+https://downloads.openmicroscopy.org/images/Zeiss-CZI/idr0011/readme.txt))
+from the open microscopy environment pages to your working directory.
 
-## Data Sets
+## Software Setup
 
-<!--
-FIXME: place any data you want learners to use in `episodes/data` and then use
-       a relative link ( [data zip file](data/lesson-data.zip) ) to provide a
-       link to it, replacing the example.com link.
--->
-Download the [data zip file](https://example.com/FIXME) and unzip it to your Desktop
+During these lessons we will use the [napari](https://napari.org/stable/)
+image viewer and the
+[napari-aicsimagio](https://github.com/AllenCellModeling/napari-aicsimageio)
+ plugin to load different image types.
+For beginners we recommend using Anaconda to install and manage these
+packages. The first step is to
+[download and install Anaconda](https://www.anaconda.com/download#downloads).
 
-## Software Setup
+The remaining instructions are written assuming you will be installing and
+running the software from a terminal. To open a terminal follow the
+instructions for your operating system in the
+['Starting Conda' section of the Anaconda getting started pages](
+https://conda.io/projects/conda/en/latest/user-guide/getting-started.html#starting-conda).
+Run the following commands in the terminal (lines starting
+with # are comments and do not need to be run).
+You can run the commands by copy and pasting them into the terminal and
+pressing the <kbd>Enter</kbd>  key.
+
+```bash
+# Make sure conda is up to date
+conda update -n base conda
+
+# We need to use pip to install the napari-aicsimagio plugin, so configure
+# conda to work well with pip
+conda config --set pip_interop_enabled True
+
+# Create a virtual environment (napari-env) for the lessons
+conda create -y -n napari-env -c conda-forge python=3.9 bioformats_jar
+
+# Activate the napari-env virtual environment
+conda activate napari-env
+
+# Install napari and plugins using pip
+pip install "napari[all]" napari-aicsimageio
+```
+
+## Post Setup Checks
+
+Before starting the course it is important to check that your setup is working.
+
+::::::::::::::::::::::::::::::::::::::: challenge
+
+## Check napari opens a tiff image.
+
+Any napari installation should be able to open a tiff image. Start napari.
+Note that if you're already in the napari-env virtual environment from the
+installation steps it is not necessary to run `conda activate napari-env`.
+```bash
+conda activate napari-env
```

Comment:
Perhaps we should tell people that they should see `(napari-env)` on the left hand side of the terminal if the environment is activated?

Reply:
I decided not to do that as it's more information than they strictly need. Running `conda activate napari-env` from within the virtual environment has no negative effects, so they'll either know they're in the environment (more advanced users) and wonder why we keep asking them to activate it, or won't know and just keep running `conda activate napari-env` with no ill effects. 
That was my logic anyway, happy to add a note about the command prompt though if you prefer.

Reply:
It was more to indicate that the activation was a success. What i had in mind is they may not know that their creation of an environment failed or not if they are not familiar with a terminal (although I guess they would get an Error message saying that environment does not exist.

Reply:
OK, I think that makes sense. I've added a note about the prompt in the script instructions. c4b86da